demo/index.tsx

```diff
@@ -45,7 +45,7 @@ export default class Demo extends React.Component {
 
     return (
       <MuiThemeProvider theme={theme}>
-        <GraphQLVoyager introspection={introspection}>
+        <GraphQLVoyager fetcher={introspection}>
```

Comment:
`introspection` can be an JSON object here.
Can you please test it with `SDL` and `Introspection` tabs?